test/argent-account-test.ts

```diff
@@ -44,10 +44,13 @@ describe("Argent account", function () {
         });
         argentAccountAddress = account.address;
 
+        const estimatedFee = await account.estimateDeployAccountFee();
+        console.log("Estimated deploy account fee: ", estimatedFee);
+
         await mint(account.address, 1e18);
         console.log("Funded account");
 
-        const deploymentTxHash = await account.deployAccount({ maxFee: 1e18 });
```

Comment:
Ok, maxFee is no longer hardcoded, but shouldn't we do that in other places in this file?

Reply:
Yes, I can add a new test block in this file.